crates/uv-requirements/src/specification.rs

```diff
@@ -109,14 +109,6 @@ impl RequirementsSpecification {
                 }
             }
             RequirementsSource::RequirementsTxt(path) => {
-                if !(path == Path::new("-")
-                    || path.starts_with("http://")
-                    || path.starts_with("https://")
-                    || path.is_file())
-                {
-                    return Err(anyhow::anyhow!("File not found: `{}`", path.user_display()));
-                }
```

Comment:
@BurntSushi - The test we want here is: can we `fs::read` this? But it's fine, we just error later (see changed fixture).